src/views/BioPage.jsx

```diff
@@ -1,8 +1,102 @@
 // LIBRARY IMPORTS
 import React from 'react';
-
+import { useParams, Link } from 'react-router-dom';
 // LOCAL IMPORTS
+import { bioData } from '../assets/bioData.js';
```

Comment:
We want to use the `fullDataSet` ref created in `App.jsx` and pass it as a prop instead of importing the static file (while right now the data is the same either way, this helps with app scalability)